.doc/connecting.asciidoc

```diff
@@ -6,15 +6,186 @@ This page contains the information you need to connect and use the Client with
 
 **On this page**
 
-* <<auth-reference, Authentication options>>
+* Connecting
+** <<connecting-to-elastic-cloud, Connecting to Elastic Cloud>>
+** <<connecting-to-self-managed, Connecting to a self-managed cluster>>
+** <<verifying-with-ca, Verifying HTTPS with CA certificates>>
+** <<verifying-with-fingerprint, Verifying HTTPS with certificate fingerprint>>
+** <<connecting-without-security, Connecting without security enabled>>
+** <<connecting-multiple-nodes, Connecting to multiple nodes>>
+* <<auth-reference, Authentication>>
+** <<auth-basic, Basic Authentication>>
+** <<auth-token, HTTP Bearer authentication>>
+* <<compatibility-mode, Compatibility mode>>
 * <<client-usage, Using the client>>
+* <<connecting-faas, Using the Client in a Function-as-a-Service Environment>>
+
+[discrete]
+[[connecting-to-elastic-cloud]]
+==== Connecting to Elastic Cloud
+
+If you are using https://www.elastic.co/cloud[Elastic Cloud], the client offers
+an easy way to connect to it. You must pass the Cloud ID that you can find in
+the cloud console and the corresponding API key.
+
+[source,go]
+------------------------------------
+cfg := elasticsearch.Config{
+        CloudID: "CLOUD_ID",
+        APIKey: "API_KEY"
+}
+es, err := elasticsearch.NewClient(cfg)
+------------------------------------
+**IMPORTANT**: you need to copy and store the `API key` in a secure place, since you will not be able to view it again in Elastic Cloud.
```

Comment:
If we go without the syntax, the output will be rendered as an admonition box.
```suggestion
IMPORTANT: you need to copy and store the `API key` in a secure place since you will not be able to view it again in Elastic Cloud.
```

Reply:
Oh awesome, thank you!